quizcon/templates/main/lti_assignment.html

```diff
@@ -4,49 +4,120 @@
 {% block title %}{{quiz.title|truncatewords:3}}{% endblock %}
 
 {% load coursetags static %}
-
+{% block topnavbar %}
+<nav class="navbar navbar-expand-lg navbar-dark">
+    <div class="container-fluid">
+        <a class="navbar-brand" href="/course/1" title="Home"><strong><span class="text-light">QuizCon</span></strong> </a>
+    </div>
+</nav>
+{% endblock %}
 {% block content %}
 <div class="quiz-container">
-    <h1>
-        {{quiz.title}}
-    </h1>
     {% if is_faculty %}
-        <p>Instructors cannot submit an assignment</p>
+        <div class="alert alert-warning" role="alert">
+            This is a preview of what your students will see.
```

Comment:
I'm wondering if we want to add meta-information about the quiz here? Perhaps include:
* Questions are randomized. (if checked)
* Question answers are always randomized.
* Show answer information, e.g. Students will be able to see their answers after X date, etc.